src/components/staking/PageValidator.vue

```diff
@@ -230,22 +238,11 @@ export default {
       `liquidAtoms`,
       `connected`
     ]),
-    validator() {
-      const validator = this.delegates.delegates.find(
-        v => this.$route.params.validator === v.operator_address
-      )
-      if (validator) {
-        validator.signing_info = this.delegates.signingInfos[
-          validator.operator_address
-        ]
-      }
-
-      return validator
-    },
     selfBond() {
       return percent(this.delegates.selfBond[this.validator.operator_address])
     },
     uptime() {
+      // We no longer have missed bloks to calculate this
```

Comment:
? @thebkr7 do we calculate the uptime already directly?

Reply:
Yes. We will have it saved directly so it will be fetched with the rest of the validators information. It is an additional query on the backend side because it is not included in all the validators information.

Reply:
@thebkr7 When will this information be available?